platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -109,21 +116,17 @@
 public class MerkleStateRoot extends PartialNaryMerkleInternal
         implements MerkleInternal, SwirldState, State, MerkleRoot {
 
-    // PlatformState is a special case, and it always has 0 index
-    static final int PLATFORM_STATE_INDEX = 0;
     private static final Logger logger = LogManager.getLogger(MerkleStateRoot.class);
 
     /**
      * Used when asked for a service's readable states that we don't have
      */
     private static final ReadableStates EMPTY_READABLE_STATES = new EmptyReadableStates();
 
-    //    private static final long CLASS_ID = 0x2de3ead3caf06392L;
-    // Uncomment the following class ID to run a mono -> modular state migration
-    // NOTE: also change class ID of ServicesState
     private static final long CLASS_ID = 0x8e300b0dfdafbb1aL;
-    private static final int VERSION_1 = 30;
-    private static final int CURRENT_VERSION = VERSION_1;
+    // Migrates from `PlatformState` to State API singleton
+    public static final int VERSION_2 = 31;
+    private static final int CURRENT_VERSION = VERSION_2;
```

Comment:
Just checking: we don't need to update the `CLASS_ID` too, do we?

Reply:
`CLASS_ID` should remain intact.